test/jdk/java/foreign/TestCopyFrom.java

```diff
@@ -55,7 +56,7 @@ public void testByteCopy(SegmentSlice s1, SegmentSlice s2) {
             Type.BYTE.set(s1, i, i);
         }
         //perform copy
-        s2.segment.copyFrom(s1.segment.asSlice(0, size));
+        MemorySegment.copy(s1.segment, 0, s2.segment, 0, size);
```

Comment:
Looks like this could use the simpler overload:

```suggestion
        MemorySegment.copy(s1.segment, s2.segment, size);
```